frontend/pages/model/[modelId]/access-request/[accessRequestId]/review.tsx

```diff
@@ -0,0 +1,142 @@
+import { ArrowBack } from '@mui/icons-material'
+import { Button, Card, Container, Divider, Grid, Paper, Stack, Typography } from '@mui/material'
+import { useGetAccessRequest, useGetAccessRequestsForModelId } from 'actions/accessRequest'
+import { useGetModel } from 'actions/model'
+import { postReviewResponse, useGetReviewRequestsForModel } from 'actions/review'
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+import Loading from 'src/common/Loading'
+import ReviewWithComment, { ResponseTypeKeys } from 'src/common/ReviewWithComment'
+import UserDisplay from 'src/common/UserDisplay'
+import MultipleErrorWrapper from 'src/errors/MultipleErrorWrapper'
+import Link from 'src/Link'
+import MessageAlert from 'src/MessageAlert'
+import Wrapper from 'src/Wrapper'
+import { mutate } from 'swr'
+import { formatDateString } from 'utils/dateUtils'
+import { getErrorMessage } from 'utils/fetcher'
+
+export default function AccessRequestReview() {
+  const router = useRouter()
+  const { modelId, accessRequestId }: { modelId?: string; accessRequestId?: string } = router.query
+
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const { model, isModelLoading, isModelError } = useGetModel(modelId)
+  const { accessRequest, isAccessRequestLoading, isAccessRequestError } = useGetAccessRequest(modelId, accessRequestId)
+  const { mutateAccessRequests } = useGetAccessRequestsForModelId(modelId)
+  const { mutateReviews } = useGetReviewRequestsForModel({
+    modelId,
+    accessRequestId: accessRequestId as string,
```

Comment:
Instead of asserting the type you can do the following to ensure it's a string, even if undefined
```
accessRequestId: `${accessRequestId}`
```